lib/listcoef.gi

```diff
@@ -262,73 +262,50 @@ end );
 
 #############################################################################
 ##
-#M  MultRowVector( <list1>, <poss1>, <list2>, <poss2>, <mult> )
+#M  MultVectorLeft( <list>, <mul> )
 ##
-InstallMethod( MultRowVector,"generic method",
+InstallMethod( MultVectorLeft,
```

Comment:
If `MultVectorLeft` is just a synonym, it would be more natural to install the method for `MultVector` here...